package.json

```diff
@@ -63,6 +63,7 @@
     "debug": "^3.1.0",
     "ethers": "^4.0.27",
     "eventemitter3": "^3.0.1",
+    "node-cache": "^4.2.0",
```

Comment:
Note `node-cache` depends on `lodash`, would be interested how much this affects the browser bundle size.

Reply:
821KB on `master` --> 844KB on this branch

Reply:
Oh man ok it's huge already